knowledge-base/chart-localized-labels.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Localize numeric labels in the Chart
+description: How to localize numeric labels in the Chart
+type: how-to
+page_title: Localize numeric labels in the Chart
+slug: chart-kb-localized-numeric-labels
+position:
+tags:
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Chart for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I am having a Chart and I would like my labels to be localization aware.
+
+
+## Solution
+
+If you require your label to be a number only, you can use the [`Format`]({%slug components/chart/label-template-format%}#format-strings) parameter of the `<ChartSeriesLabels>` tag. It takes a [standard numeric format strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings) and is localization aware.
+
+If you want to expose more information beyond a number, for example, some text, you can utilize the [`Template`]({%slug components/chart/label-template-format%}#templates) parameter, which is exposed from the `<ChartSeriesLabels>` tag. In a JavaScript file implement the desired number formatting by using the `toLocaleString` JavaScript method. An exemplary implementation can be seen in the example below:
+
+### Step by step explanation
+
+1. Use a custom [template]({%slug components/chart/label-template-format%}#templates)
+
+2. Implement the desired number formatting function in a JavaScript file (in the example below, we will call it `template-helpers.js` and it resides in the `wwwroot` folder).
+
+3. Reference that file in your root component (`_Host.cshtml` for a server-side app, or `index.html` for a client-side app).
+
+4. Call the custom formatting function from the template and pass the needed arguments to it. It must return the string you want shown in the template.
+
+#### Localize numeric labels in the Chart using the Template parameter
+
+````Razor
+@* This example shows how to localize numeric labels in the Chart using the Template parameter *@
+
+@*used to get the current culture*@
+@using System.Threading
+
+<TelerikChart>
+
+    <ChartLegend Visible="true" Position="ChartLegendPosition.Top"></ChartLegend>
+
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Donut"
+                     Data="@this.ChartData"
+                     Field="@nameof(Data.Value)"
+                     CategoryField="@nameof(Data.XValue)"
+                     ColorField="@nameof(Data.BackgroundColor)"
+                     StartAngle="270">
+            <ChartSeriesLabels Position="ChartSeriesLabelsPosition.OutsideEnd"
+                               Visible="true"
+                               Template="@GetFormattedNumericValue()">
+            </ChartSeriesLabels>
+        </ChartSeries>
+    </ChartSeriesItems>
+
+</TelerikChart>
+
+@code {
+    public class Data
+    {
+        public string XValue { get; set; }
+
```

Comment:
You can remove these new lines. Consider putting the code for the template first, and maybe refactoring it into  property, something like `LocalizedTemplateString => GetFormattedNumericValue()` so the chart markup is cleaner